atomic-write.cabal

```diff
@@ -43,7 +43,7 @@ license:             MIT
 license-file:        LICENSE
 author:              Justin Leitgeb
 maintainer:          justin@stackbuilders.com
-copyright:           2015 Stack Builders Inc.
+copyright:           2015-2017 Stack Builders Inc.
```

Comment:
@acamino, @jsl not sure if it is a big deal. But the copyright in the cabal file didn't match with the one in the `LICENSE` file.